src/components/Checkbox/CoreuiCheckBox.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { CoreuiCheckBox } from './CoreuiCheckBox';
+import { FormikFormWrapper } from '../FormikFormWrapper/FormikFormWrapper';
+
+it('should show CoreuiCheckBox', () => {
+  const wrapper = shallow(<CoreuiCheckBox name="bar" id="id-foo" label="foo" />);
+  expect(wrapper.find('#id-foo').length).toBe(1);
+  expect(wrapper.find('.form-control').length).toBe(1);
+});
+
+it('should implement props', () => {
+  const wrapper = shallow(
+    <CoreuiCheckBox
+      name="bar"
+      className="classname-foo"
+      label="foo"
+    />,
+  );
+  expect(wrapper.find('#bar-input').length).toBe(1);
+  expect(wrapper.find('.classname-foo').length).toBe(1);
+  expect(wrapper.find('[name="bar"]').length).toBe(1);
+});
+
+it('should change initial prop', () => {
```

Comment:
This test description is invalid. There is not prop tests in this test suite. This test ONLY test if change simulation working correctly